src/mame/drivers/galaxian.cpp

```diff
@@ -5309,6 +5327,56 @@ static INPUT_PORTS_START( scramble )
 	PORT_BIT( 0xff, 0x00, IPT_UNUSED )
 INPUT_PORTS_END
 
+// unknown button config, button 1 moves up the screen, with up being jump and down being drop item
+// using standard mapping, but maybe that is correct?
```

Comment:
Galaxian HW has interchangeable input pinouts for Up and Fire cfr. [here](https://www.mikesarcade.com/cgi-bin/spies.cgi?action=url&type=pinout&page=Pac.html)
If I've understood what the game is correctly it makes more sense the other way around?

Reply:
I don't know, it seems easier to play with up as jump and down as drop bomb, moving up the screen is more like accelerate, connected to a button, I *think* it's correct as is.

Reply:
I totally don't expect jump to be up on a racing game. 
And to make a comparison: Bump N Jump doesn't map jump to up.

Reply:
Bump N Jump is a different style of game.

Reply:
Both are overhead racing games of the early 80s, and BnJ maps up as acceleration.

Reply:
ShouTime had the instruction / wiring sheet anyway, Up is jump, Down is release Item, A is accelerate http://mamedev.emulab.it/haze/reference/namenayo_instructions.jpg so MAME is correct

Reply:
I don't see Up clearly written there.

Reply:
Then you can't read.

Reply:
If it's confirmed then update the PORT_NAME labels reflecting it (it still says `// move up screen` which is confusing as hell).